interpreter/interpreter.go

```diff
@@ -487,27 +487,51 @@ func (itrp *Interpreter) doArray(ctx context.Context, a *semantic.ArrayExpressio
 }
 
 func (itrp *Interpreter) doObject(ctx context.Context, m *semantic.ObjectExpression, scope values.Scope) (values.Value, error) {
-	obj := values.NewObject()
-	if m.With != nil {
-		with, err := itrp.doExpression(ctx, m.With, scope)
-		if err != nil {
-			return nil, err
-		}
-		with.Object().Range(func(k string, v values.Value) {
-			obj.Set(k, v)
-		})
+	if label, nok := itrp.checkForDuplicates(m.Properties); nok {
```

Comment:
What is this check for?

Reply:
I guess right now, we won't allow duplicate keys?

Reply:
That seemed to be the old behavior.